### PR TITLE
Add Film Type & Species Type

### PR DESCRIPTION
**Pull request by xuorig**
cc @cjoudrey

### CODE REVIEW

**Comment by xuorig** [comment]
Added a big controller test just so we know we dont break anything as we yolo ship

File: test/controllers/graphql_controller_test.rb [hunk 1]
```diff
@@ -0,0 +1,174 @@
+class GraphQLControllerTest < ActionDispatch::IntegrationTest
+  test "should execute graphql queries" do
```
Comment (cjoudrey) [inline test/controllers/graphql_controller_test.rb]: :fire: `should` :trollface:

Comment (xuorig) [reply test/controllers/graphql_controller_test.rb]: lool damn

File: app/models/graph/types/film.rb [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+
+      field :title, types.String, "The title of this film"
+      field :episodeID, types.Int, "The episode number of this film.", property: :episode_id
+      field :openingCrawl, types.String,
+        "The opening paragraphs at the beginning of this film.", property: :opening_crawl
+
+      field :director, types.String, "The name of the director of this film."
+      field :producers, types[types.String] do
+        description "The name(s) of the producer(s) of this film."
+        resolve ->(film, _, _) { film.producer.delete(" \t\r\n").split(",") }
```
Comment (cjoudrey) [inline app/models/graph/types/film.rb]: Just curious, why did you need the `delete(" \t\r\n")`? Could we not have just done `.split(", ")`?

Comment (xuorig) [reply app/models/graph/types/film.rb]: i was afraid of shitty data but i think it might be over kill, youre right

File: app/models/graph/types/species.rb [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+      field :averageLifespan, types.Int do
+        description "The average lifespan of this species in years, null if unknown."
+        property :average_lifespan
+      end
+
+      field :eyeColors, types[types.String] do
+        description "Common eye colors for this species, null if this species does not typically have eyes."
+        resolve ->(species, _, _) do
+          return unless colors = species.eye_colors
+          return if species.eye_colors.include?("none")
```
Comment (cjoudrey) [inline app/models/graph/types/species.rb]: If we don't care about `"none"`, I'll fix the import script to simply omit this value so we can :fire: this line.

Comment (xuorig) [reply app/models/graph/types/species.rb]: 👍

Comment (xuorig) [reply app/models/graph/types/species.rb]: maybe we can 🔥 `n/a` too and add that to the description

Comment (xuorig) [reply app/models/graph/types/species.rb]: right now there are weird cases with `['n/a']`

File: app/models/graph/types/film.rb (hunk 2 above)
Comment (cjoudrey) [reply app/models/graph/types/film.rb]: Yeah, I wouldn't worry about this. We can always fix the data when importing if this is an issue.

**Comment by xuorig** [comment]
Going to merge this if you dont have major concerns, we can fix as we go
